app/views/orders/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container transparent">
+  <h1>Orders:</h1>
+
+  <table class="centered bordered responsive-table">
+    <thead>
+      <tr>
+          <th data-field="id">Order ID</th>
+          <th data-field="status">Status</th>
+          <th data-field="view">View</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @orders.each do |order| %>
+      <tr>
+        <td><%= order.id %></td>
+        <td><%= order.status.capitalize %></td>
```

Comment:
Should we move the capitalize method into the model? Will we need the order status non-capitalized? Maybe for...emails to the end user about their order status?